.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml

```diff
@@ -0,0 +1,138 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+
+data:
+  app-config-rhdh.yaml: |
+    app:
+      title: Red Hat Developer Hub
+      analytics:
+        segment:
+          testMode: true
+          writeKey: temp
+    backend:
+      auth:
+        keys:
+          - secret: temp
+    integrations:
+      # Plugin: GitHub
+      github:
+        - host: github.com
+          apps:
+            - appId: ${GITHUB_APP_APP_ID}
+              clientId: ${GITHUB_APP_CLIENT_ID}
+              clientSecret: ${GITHUB_APP_CLIENT_SECRET}
+              webhookUrl: ${GITHUB_APP_WEBHOOK_URL}
+              webhookSecret: ${GITHUB_APP_WEBHOOK_SECRET}
+              privateKey: |
+                ${GITHUB_APP_PRIVATE_KEY}
+      bitbucketServer:
+        - host: bitbucket.com
+          apiBaseUrl: temp
+          username: temp
+          password: temp
+      gitlab:
+        - host: gitlab.com
+          token: temp
+    auth:
+      # see https://backstage.io/docs/auth/ to learn about auth providers
+      environment: development
+      providers:
+        # Plugin: GitHub
+        github:
+          development:
+            clientId: ${GITHUB_APP_CLIENT_ID}
+            clientSecret: ${GITHUB_APP_CLIENT_SECRET}
+
+    proxy:
+      skipInvalidProxies: true
+      # endpoints: {}
+      endpoints: 
+        # Other Proxies
+        # customize developer hub instance
+        '/developer-hub':
+          target: http://test-backstage-customization-provider-rhdh.backstage-os-2-eu-de-2-bx2-c74b3ed44ce86949f501aefb2db80652-0000.eu-de.containers.appdomain.cloud/
+          changeOrigin: true
+          # Change to "false" in case of using self hosted cluster with a self-signed certificate
+          secure: true   
+        '/developer-hub/tech-radar':
+          target: http://test-backstage-customization-provider-rhdh.backstage-os-2-eu-de-2-bx2-c74b3ed44ce86949f501aefb2db80652-0000.eu-de.containers.appdomain.cloud/tech-radar
```

Comment:
Should the `target`s be hardcoded?

Reply:
@schultzp2020 good catch Paul. @josephca lets not hardcode the targets. In addition to OCP and k8s in IBM Cloud we will need to test on EKS, AKS, and GKE.

Reply:
@schultzp2020 Nope. I don't like the hardcoded code. :) Fixed.